include/oneapi/mkl.hpp

```diff
@@ -23,5 +23,6 @@
 #include "oneapi/mkl/types.hpp"
 
 #include "oneapi/mkl/blas/blas.hpp"
+#include "oneapi/mkl/rng/rng.hpp"
```

Comment:
`rng.hpp` should be on the same level as `rng` folder to avoid this duplication in the include line. BLAS also should be updated, but I guess we will do it after PR 43

Reply:
ok, moved to oneapi/mkl/rng.hpp

Do I need to move backend_selector.hpp, backends.hpp from  oneapi/mkl/detail/ to oneapi/mkl as they are in namespace oneapi::mkl::?